lua/grpc-mission.lua

```diff
@@ -15,18 +15,19 @@ end
 -- Let DCS know where to find the DLLs
 package.cpath = package.cpath .. GRPC.dllPath .. [[?.dll;]]
 
+env.info("[GRPC] Writing " .. lfs.writedir() .. [[Data\dcs-grpc.lua]] )
 -- Make settings available to gRPC hook
-local file, err = io.open(lfs.writedir() .. [[Data\dcs-grpc.lua]], "w")
-if err then
-	env.error("[GRPC] Error writing config")
-else
-	file:write(
+local f, err = io.open(lfs.writedir() .. [[Data\dcs-grpc.lua]], "w")
+if f then
```

Comment:
```suggestion
local file, err = io.open(lfs.writedir() .. [[Data\dcs-grpc.lua]], "w")
if file then
```

or rename `file` below to `f` 🙂

Reply:
Thanks, that is what I get for early morning commits :cry: